src/main/kotlin/racing/domain/Capacity.kt

```diff
@@ -15,7 +15,7 @@ value class Capacity(val value: Int) : Iterable<Int>, Comparable<Capacity> {
 
     override fun iterator(): Iterator<Int> = (0 until value).iterator()
```

Comment:
이전 [comment](https://github.com/next-step/kotlin-racingcar/pull/1071#discussion_r1224546405)에서 답변 주신 부분이네요🙂
`Capacity` 객체가 IntRange를 제공해야 한다면 프로퍼티로 제공해주면 어떨까요?
```suggestion
   val availableRange: IntRange get() = 0 until value
```

Reply:
- 프로퍼티로 제공할지 fun 함수로 제공할지 어떻게 결정해야 할지 아직 헷갈리는데, 어떤 지침이나 규칙이 있을까요? 
- Interable 을 구현해서 해당 객체가 반복가능하다고 알려주는 것 대신 IntRange를 프로퍼티로 제공할 때 어떠한 강점이 있을까요? 
- 우선 해당 내용은 반영했습니다.

Reply:
1. [코틀린 공식 문서](https://kotlinlang.org/docs/coding-conventions.html#functions-vs-properties)를 참고하시면 좋을 것 같아요.

2. `수량, 용량`은 일반적으로 반복이라는 속성이 없는 개념인 만큼 `Iterator`의 하위 타입이 되는 게 어색하다고 생각했어요. 이는 `Capacity`를 사용하는 사용자들에게 혼란을 줄 수도 있다고 생각합니다(저도 그랬구요😅).
대신 `수량만큼의 범위 제공`이라는 기능을 한정적으로 제공함으로써 사용자들이 제공자의 의도를 명확하게 파악할 수 있다는 장점이 있을 것 같아요. 마치 `Composition`을 활용한 것처럼 말이죠🙂